assemble/bin/accumulo-cluster

```diff
@@ -685,75 +514,36 @@ EOF
       ;;
     restart)
       parse_config
-      stop_all
+      ARG_ALL=1
+      control_services stop
+      control_services kill
       # Make sure the JVM has a chance to fully exit
       sleep 1
-      start_all
+      control_services start
       ;;
     start)
       parse_config
-      start_all
+      control_services start
       ;;
     stop)
       parse_config
-      stop_all
+      control_services stop
       ;;
     kill)
       parse_config
-      kill_all
+      control_services kill
       ;;
     start-here)
       parse_config
-      start_here
+      control_services start --local --all
```

Comment:
start-here and stop-here functions have been removed and they are now just shortcuts. These could be removed.